extensions/mssql/package.json

```diff
@@ -93,7 +93,21 @@
         "mssql.logDebugInfo": {
```

Comment:
is the `logDebugInfo` config still used anywhere?

Reply:
Yes it is still used by independently done UI logging.